packages/api/src/utils/logParser.ts

```diff
@@ -170,6 +171,14 @@ export const mapObjectToKeyValuePairs = (
     }
   }
 
+  if (output['string.names'].includes('db.statement')) {
```

Comment:
If it's useful, we can also check `db.system` https://opentelemetry.io/docs/specs/semconv/database/database-spans/#:~:text=db.system%20has%20the%20following%20list%20of%20well%2Dknown%20values.

Reply:
Yep thought about adding that as well 👍 if we want to support specific DBs we can do that, I thought for now since we're adding a new key it can be completely general.